### PR TITLE
Promisifying

### PR DESCRIPTION
**Pull request by agrampa**

### CODE REVIEW

File: lab-allie/lib/storage.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      return JSON.parse(data);
+    });
+    console.log('json.parse data', JSON.parse(data));
+    return JSON.parse(data);
+  })
+  .catch(console.error('error in fs.readFileProm'));
+};
+
+exports.fetchAll = function(schemaName) {
+  debug('#storage fetchAll');
```
Comment (jurrish) [inline lab-allie/lib/storage.js:70]: good. hit this route and fetchAll without id passed in. I've been seeing students pass in a single id as "access" to all created instances.

File: lab-allie/test/server-test.js [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+          done();
+        });
+      });
+    });
+  });
+  
+  describe('GET method', function() {
+    let testGet;
+    before(done => {
+      chai.request(server)
```
Comment (jurrish) [inline lab-allie/test/server-test.js:89]: good setup.

File: lab-allie/test/server-test.js [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+        chai.request(server)
+        .del(`/api/album?id=${testDelete.id}`)
+        .end((err, res) => {
+          if (err) console.error(err);
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+      
+      it('should return an error on a bad request', done => {
```
Comment (jurrish) [inline lab-allie/test/server-test.js:286]: how could you test for a 204, the proper response for a Delete operation?

Comment (agrampa) [reply lab-allie/test/server-test.js:286]: mine are returning 404, not found

**Pull request merged**